src/parser/jobs/sam/modules/index.js

```diff
@@ -2,20 +2,24 @@ import {AoeChecker} from './AoeChecker'
 import {Buffs} from './Buffs'
 import {Combos} from './Combos'
 import {Fuka} from './Fuka'
-// import {Higanbana} from './Higanbana'
 import Interrupts from './Interrupts'
+import {Kenki} from './Kenki'
+// import {Higanbana} from './Higanbana'
```

Comment:
commented out so it doesn't really matter here, but this should still be above Interrupts to maintain alphabetical order of imports